AmericanMedicalAssociation/AMA-style-guide
# EWL 3269 - Subcategory topics with description

Ticket: https://issues.ama-assn.org/browse/EWL-3269

Created subcategory topics with description component

To test:
- Go to Molecules > Components > Subcategory topics with Description
- Check component at all viewports to ensure it doesn't break
- Check hover-states of links
